libexec/bootstrap-scripts/deffile-driver-localimage.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# 
+# Copyright (c) 2017, SingularityWare, LLC. All rights reserved.
+#
+# This software is licensed under a 3-clause BSD license.  Please
+# consult LICENSE file distributed with the sources of this project regarding
+# your rights to use or distribute this software. 
+#
+# 
+
+## Basic sanity
+if [ -z "$SINGULARITY_libexecdir" ]; then
+    echo "Could not identify the Singularity libexecdir."
+    exit 1
+fi
+
+## Load functions
+if [ -f "$SINGULARITY_libexecdir/singularity/functions" ]; then
+    . "$SINGULARITY_libexecdir/singularity/functions"
+else
+    echo "Error loading functions: $SINGULARITY_libexecdir/singularity/functions"
+    exit 1
+fi
+
+if [ -z "${SINGULARITY_ROOTFS:-}" ]; then
+    message ERROR "Singularity root file system not defined\n"
+    exit 1
+fi
+
+if [ -z "${FROM:-}" ]; then
+    message ERROR "Required Definition tag 'From:' not defined.\n"
+    exit 1
+fi
+
+if [ ! -f "${FROM:-}" ]; then
+    message ERROR "${FROM} does not exist\n"
+    exit 1
+fi
+
+if [ -z "${SINGULARITY_BUILDDEF:-}" ]; then
+    exit
+fi
```

Comment:
This conditional exists for legacy reasons, and can probably be removed (in both sources) and/or abort. This check occurs earlier and will trigger the `main-null.sh` invoker instead of `main-deffile.sh` (meaning that this driver should never get invoked).